draft-kuepau-iab-privacy-partitioning.md

```diff
@@ -430,60 +437,121 @@ token issuance servers.
 ~~~
 {: #diagram-privacypass title="Diagram of contexts in Privacy Pass"}
 
-## DAP (PPM)
+## Privacy Preserving Measurement
 
-Distributed Aggregation Protocol (DAP) is a protocol being developed in the PPM working group to allow
-servers to collect aggregate metrics across a large population of clients, without learning individual
-data points about clients.
+The Privacy Preserving Measurement (PPM) working group is chartered to develop protocols and systems
+that help a data aggregation or collection server (or multiple, non-colluding servers) compute aggregate
+values without learning the value of any one client's individual measurement. Distributed Aggregation
+Protocol (DAP) is the primary working item of the group.
 
-TODO: Analysis of contexts
+At a high level, DAP uses a combination of cryptographic protection (in the form of secret sharing amongst
+non-colluding servers) to establish two contexts: an "upload context" between clients and non-colluding
+aggregation servers wherein aggregation servers possibly learn client identity but nothing about their individual
+measurement reports, and a "collect context" wherein a collector learns aggregate measurement results and nothing
+about individual client data.
 
-TODO: Diagram for DAP
+~~~ aasvg
++-------------------------------------+--------------------+
+| Upload Context                      | Collect Context    |
+|                     +------------+  |                    |
+|              +------>   Helper   |  |                    |
+| +--------+   |      +------^-----+  |                    |
+| |        +---+             |        |   +-----------+    |
+| | Client |                 |        |   | Collector |    |
+| |        +---+             |        |   +-----+-----+    |
+| +--------+   |      +------V-----+  |         |          |
+|              +------>   Leader   <------------+          |
+|                     +------------+  |                    |
++-------------------------------------+--------------------+
+~~~
+{: #pa-topology title="Diagram of contexts in DAP"}
 
-# Analysis of Privacy Partioning
```

Comment:
I'd still like to have a section for analysis, and not _just_ limits. I quite like the limits section, but I think there's still room for some general comments about "as new protocols come out, here's a guide to think about their use of partitioning".

So, could we add the stub back for now?

Reply:
I added back a stub with a slightly different title to better match what you have in mind. Have a look?